examples/C/Hash/GettingStarted.c

```diff
@@ -45,7 +45,7 @@ This example is available in full on [GitHub](https://github.com/51Degrees/devic
 #include "../../../src/hash/hash.h"
 #include "../../../src/hash/fiftyone.h"
 
-#define MAX_EVIDENCE 5
+#define MAX_EVIDENCE 6

```

Comment:
looks like we stopped using this constant and instead hardcode the number of evidence?

Reply:
It is still used in `typedef`.
- https://github.com/51Degrees/device-detection-cxx/pull/73/files/ced48c8d1e2b221553e3c605476ae79903af3c39#diff-f997f3320decacbc56880b99e83ff7fbe43349bfcb60d4156d4875ff1d963715R65-R71